src/services/API/Plio.js

```diff
@@ -85,14 +90,30 @@ export default {
 
   duplicatePlio(plioId) {
     // create a clone of plioId plio
-    return apiClient().post(pliosEndpoint + plioId + "/duplicate");
+    return apiClient().post(pliosEndpoint + plioId + duplicatePlioEndpoint);
   },
 
   async getUniqueUsersCount(plioId) {
     // get the count of unique users who watched the given plio
-    var response = await apiClient().get(
-      pliosEndpoint + plioId + "/unique_users"
-    );
-    return response.data;
+    // refer to this example: https://cube.dev/blog/vue-dashboard-tutorial-using-cubejs/
+
+    // this is the query in the CubeJS Data Schema which is automatically
+    // converted to SQL by CubeJS: https://cube.dev/docs/getting-started-cubejs-schema
+    var uniqueUsersQuery = {
+      measures: ["Session.uniqueUsers"],
+      filters: [
+        {
+          member: "Plio.uuid",
+          operator: "equals",
+          values: [plioId],
+        },
+      ],
+    };
```

Comment:
Before this cube js change, we were querying the plio ka endpoint to get this metric. But here we're querying sessions ka cube, why is that? can't it be done by querying the plio ka cube?